public/src/components/common/nav_btn_group.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { ButtonGroup, Button } from 'react-bootstrap';
+
+export default class DashboardButtonGroup extends Component {
+  constructor(props) {
+    super(props);
+    this.handleQuestionsClick = this.handleQuestionsClick.bind(this);
+    this.handleCampaignsClick = this.handleCampaignsClick.bind(this);
+    this.handleScriptsClick = this.handleScriptsClick.bind(this);
+    this.handleContactListsClick = this.handleContactListsClick.bind(this);
+    this.handleAvailCampaignsClick = this.handleAvailCampaignsClick.bind(this);
+    this.handleJoinedCampaignsClick = this.handleJoinedCampaignsClick.bind(this);
+  }
+
+  handleQuestionsClick() {
+    this.props.history.push('/admin/questions');
+  }
+  handleCampaignsClick() {
+    this.props.history.push('/admin/campaigns');
+  }
+  handleScriptsClick() {
+    this.props.history.push('/admin/scripts');
+  }
+  handleContactListsClick() {
+    this.props.history.push('/admin/contactLists');
+  }
+  handleAvailCampaignsClick() {
+    this.props.history.push('/volunteers/campaigns');
+  }
+  handleJoinedCampaignsClick() {
+    this.props.history.push('/volunteers/campaigns/mine');
```

Comment:
I wonder if there's a better path name for this. /mycampaigns? or /dashboard? I can also explore this more in my ticket if you think that's best

Reply:
I think since you're building out that view you can change the route. It's really just more of a placeholder for the tab.